docs(multitenancy): Add Warning about scheduledRefreshContexts (#1923)
cube-js/cube.js: docs/content/Configuration/Config.md

};
```

Learn more about [scheduled refresh here](caching#keeping-cache-up-to-date)

Best practice is to run `scheduledRefreshTimer` in a separate worker Cube.js
instance. For serverless deployments [REST API](rest-api#api-reference-v-1-run-scheduled-refresh) should be used
instead of timer.

> There are some options that are important for the scheduled refresh:
> 
> - [`scheduledRefreshTimeZones`](#options-reference-scheduled-refresh-time-zones)
> - [`scheduledRefreshContexts`](#options-reference-scheduled-refresh-contexts)
> 

Reviewer: ```suggestion
You may also need to configure [`scheduledRefreshTimeZones`](#options-reference-scheduled-refresh-time-zones) and [`scheduledRefreshContexts`](#options-reference-scheduled-refresh-contexts).
```
